packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -201,7 +202,7 @@ declare module '@docusaurus/plugin-content-blog' {
     /**
      * Whether the truncate marker exists in the post's content.
      */
-    readonly truncated?: boolean;
+    readonly truncateMarker: boolean;
```

Comment:
What about `hasTruncateMarker`? This sounds like a string.

Reply:
agree, will rename it

also the plugin option is already "truncateMarker" and is a Regex